src/h5web/vis-packs/core/heatmap/Mesh.tsx

```diff
@@ -45,7 +45,7 @@ function Mesh(props: Props): ReactElement {
     <>
       {material && (
         <mesh material={material}>
-          <planeBufferGeometry attach="geometry" args={[width, height]} />
+          <planeGeometry attach="geometry" args={[width, height]} />
```

Comment:
Turns out that `PlaceBufferGeometry` is just an alias. `PlaneGeometry` is already a `BufferGeometry`.